SableCC/Generated/GGcodeGrammar.sablecc

```diff
@@ -10,47 +12,169 @@ Helpers
   letter  = lowercase | uppercase;
   idletter = letter | '_';
   idchar  = letter | '_' | digit;
+  xyz = 'x' | 'y' | 'z' | 'X' | 'Y' | 'Z';
```

Comment:
We shouldnt allow both small and big characters for xyz

Reply:
It has been fixed